### PR TITLE
Add --version option

### PR DESCRIPTION
**Pull request by daniel-larraz**
This enables users to easily check the version of the program they are using.

### CODE REVIEW

File: src/repairwheel/__init__.py [hunk 1]
```diff
@@ -3,3 +3,5 @@
 """
 repairwheel
 """
+
+__version__ = "0.2.9"
```
Comment (jvolkman) [inline src/repairwheel/__init__.py:7]: So this is what gets updated when cutting a new release, rather than `pyproject.toml`?

Comment (daniel-larraz) [reply src/repairwheel/__init__.py:7]: Correct.

Comment (daniel-larraz) [reply src/repairwheel/__init__.py:7]: Alternatively, you can just run `hatch version "0.3.0"`. Relevant documentation:
https://hatch.pypa.io/1.9/config/metadata/#version
https://hatch.pypa.io/1.9/version/#updating

Comment (jvolkman) [reply src/repairwheel/__init__.py:7]: Got it. Thanks.

**Pull request merged**